app/models/comment.rb

```diff
@@ -3,9 +3,13 @@ class Comment < ApplicationRecord
   attribute :text, :text
 
   # Associations
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
 
-  # Validations
-  validates :text, presence: true
+  # Methods
+  after_create :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Just like with the Post model, you should rather use `after_save` and make the `update_post_comments_counter` a private method.